### PR TITLE
Convert data types for UAGraph.nodes

### PR DESCRIPTION
**Pull request by DawidMakar-acaisoft**
The changeset consists of:

- setting correct types for `UAGraph.nodes` DataFrame
- removing leftover DataFrames in `UAGraph.from_path()` manually
- update a CSV test file

### CODE REVIEW

File: opcua_tools/nodeset_parser.py [hunk 1]
```diff
@@ -521,6 +524,54 @@ def parse_xml_without_normalization(
     }
 
 
+def get_attrib_df(nodes: pd.DataFrame) -> pd.DataFrame:
+    attrib_df = pd.DataFrame.from_records(nodes["Attrib"].values)
+
+    is_abstract_coloumn_name = "IsAbstract"
```
Comment (olavpe) [inline opcua_tools/nodeset_parser.py:530]: column*

**Comment by olavpe** [comment]
Ahh is the big improvement of using IntXX instead of python Int? and setting those datatypes that can be found in C?

File: opcua_tools/nodeset_parser.py (hunk 1 above)
Comment (DawidMakar-acaisoft) [reply opcua_tools/nodeset_parser.py]: Good catch!

**Comment by DawidMakar-acaisoft** [comment]
> Ahh is the big improvement of using IntXX instead of python Int? and setting those datatypes that can be found in C?

In general, I intended to replace `dtype=object` in the places where it can be done. It should also make a difference in which c-type integer we use. It's not a great improvement but still - noticeable.

**Pull request merged**